Classes/ViewHelpers/LinkViewHelper.php

```diff
@@ -254,7 +254,7 @@ protected function getNewsId(News $newsItem): int
                 'tx_news_domain_model_news',
                 $newsItem->getUid()
             );
-            if ($record['uid'] ?? false) {
+            if (isset($record['uid'] ?? false)) {
```

Comment:
That does not make much sense. `??` is already the fallback if uid is not set in $record. Switching to isset() makes the fallback obsolute ... this does the same in two syntax variants.

Reply:
It does not make sense to change it at all ... before and after doing the same check, just with different code variants.

Maybe the "issue" has existed before "?? false" fallback has been added ? Please retest it with the ?? fallback.

See following as demonstration why there is "no" need to change it here from ?? fallback to the isset variant:

https://3v4l.org/lu9ob

Reply:
This change is superflous. It does chage the code but do nit fix anything or chage anything in behaviour. It can be closed.

Reply:
or to make clear - $record['uid'] ?? false already covers the array key access / offset issue mentioned in the pull request message ...

See: https://3v4l.org/tU3NG

Reply:
look - you should updatethe version. You get the error because you do not have the "??" fallback. 

The issue is already fixed in the code.

You should just update to get it. Your PR came to late. That is what I mean - you are change an already existing fix for your issue to another variant, thus making it superflous.